en/Modules/language-management.md

```diff
@@ -0,0 +1,170 @@
+* Volo.Abp.LanguageManagement.Domain.Shared
+* Volo.Abp.LanguageManagement.Domain
+* Volo.Abp.LanguageManagement.Application.Contracts
+* Volo.Abp.LanguageManagement.Application
+* Volo.Abp.LanguageManagement.EntityFrameworkCore
+* Volo.Abp.LanguageManagement.MongoDB
+* Volo.Abp.LanguageManagement.HttpApi
+* Volo.Abp.LanguageManagement.HttpApi.Client
+* Volo.Abp.LanguageManagement.Web
+
+### NPM Packages
+
+* @volo/abp.ng.language-management
+* @volo/abp.ng.language-management.config
+
+## User Interface
+
+### Menu Items
+
+Language management module adds the following items to the "Main" menu, under the "Administration" menu item:
+
+* **Languages**: Language management page.
+* **Language Texts**: Language text management page.
+
+`LanguageManagementMenuNames` class has the constants for the menu item names.
+
+### Pages
+
+#### Languages
+
+Languages page is used to manage languages in the system. 
+
+![language-management-languages-page](../Images/language-management-languages-page.png)
+
+You can create a new language or edit an existing language in this page:
+
+![language-management-edit-language-modal](../Images/language-management-edit-language-modal.png)
+
+* **Enabled** languages can be selected as the system language.
+
+#### Language Texts
+
+Language texts page is used to manage texts in different languages.
+
+![language-management-language-texts-page](../Images/language-management-language-texts-page.png)
+
+You can translate a text for a language or edit the already existing translation in this page.
+
+![language-management-edit-language-text-modal](../Images/language-management-edit-language-text-modal.png)
+
+## Data Seed
+
+This module adds some initial data (see [the data seed system](https://docs.abp.io/en/abp/latest/Data-Seeding)) to the database when you run the `.DbMigrator` application:
+
+* Creates language records configured using `AbpLocalizationOptions`.
+
+If you want to change the seeded language list, see the next section.
+
+## Options
+
+### AbpLocalizationOptions
+
+`AbpLocalizationOptions` can be configured in the Domain layer, in the `ConfigureServices` method of your [module](https://docs.abp.io/en/abp/latest/Module-Development-Basics). Example:
+
+````csharp
+Configure<AbpLocalizationOptions>(options =>
+{
+    options.Languages.Add(new LanguageInfo("en", "en", "English"));
+    //Add other languages here...
+});
+````
+
+`AbpLocalizationOptions` properties:
+
+* `Resources` : Represents list of resource files used for localization in the application.
+* `GlobalContributors` :  Global localization contributors. Global contributors have precedence over other contributors.
+* `Languages` : List of languages available in the application.
+
+## Internals
+
+### Domain Layer
+
+#### Aggregates
+
+This module follows the [Entity Best Practices & Conventions](https://docs.abp.io/en/abp/latest/Best-Practices/Entities) guide.
+
+##### Language
+
+* `Language` (aggregate root): Represents a language in the system.
```

Comment:
LanguageText is also an aggregate root (will be: https://github.com/volosoft/volo/blob/dev/abp/language-management/src/Volo.Abp.LanguageManagement.Domain/Volo/Abp/LanguageManagement/LanguageText.cs)